Softeq.XToolkit.WhiteLabel.iOS/Services/StoryboardDialogsService.cs

```diff
@@ -72,6 +74,39 @@ public class StoryboardDialogsService : IDialogsService
             return dialogResult.Task;
         }
 
+        public Task<TEnum> ShowDialogAsync<TEnum>(string title,
+            string message,
+            Dictionary<TEnum, DialogOption> actions)
+            where TEnum : Enum
+        {
+            var dialogResult = new TaskCompletionSource<TEnum>();
+
+            Execute.BeginOnUIThread(() =>
```

Comment:
Implementation of this method is very much looks like implementation of the default ShowDialogAsync, but this method is more versatile. Maybe we should reuse it?
BTW, seems like default implementation of the ShowDialogAsync might be turned into extension in this case

Reply:
I agree with @TimofeyBurak, related to an issue #226